src/app/modules/people/people.router.ts

```diff
@@ -5,12 +5,17 @@ import { RouterModule, Routes } from '@angular/router';
 import { CsCoreComponent } from './cs-core/cs-core.component';
 import { SGComponent } from './sg/sg.component';
 import { AMComponent } from './am/am.component';
+import { AlumniComponent } from './alumni/alumni.component';
+import { SinglePageComponent } from './alumni/single-page/single-page.component'
 
 const peopleRoutes: Routes = [
   { path: '', redirectTo: 'cscoreteam', pathMatch: 'full'},
   { path: 'cscoreteam', component: CsCoreComponent },
   { path: 'studentguides', component: SGComponent },
-  { path: 'academicmentors', component: AMComponent }
+  { path: 'academicmentors', component: AMComponent },
+  { path: 'alumni-memoirs', redirectTo: 'alumni-memoirs/all', pathMatch: 'full' },
+  { path: 'alumni-memoirs/:id', component: AlumniComponent},
```

Comment:
So this becomes simply:
```suggestion
  { path: 'alumni-memoirs', component: AlumniComponent},
```